### ISSUE TITLE
Introduce a new item that has two start and two end times.

### ISSUE THREAD

**Issue by mcshaman** [almende/vis#497]
I have been using Timeline to map out future projects and events. Unfortunately we are not one hundred percent sure when these events will start or finish. When mapping out the timelines manually (using design software) we indicate an earliest start and a latest end. It would be great if Timeline offered this flexibility. I have mocked up an example.
![screenshot-2014-12-07-07 43 19](https://cloud.githubusercontent.com/assets/863334/5328855/5f5ebcfe-7de6-11e4-9c55-285e3f466c3d.png)

**Comment by AlexDM0** [comment]
Hi,

I saw that you asked Jos, but since I'm the one who put the files there I figured I could answer this. Anyway, while I agree it may look like those thing do not belong there, there is a reason behind it. There is much shared code between timeline and graph2d and we plan on merging them further. 

As for hijacking, it's a term commonly used when the discussion in a topic diverges from the topic itself. If you'd like to discuss this subject further, it would be clearer for people looking for answers in issues, if you make one specifically for this subject. 

Finally, even though this project is actively developed and thus in a state of flux, there is a modular system in timeline and now is as good a time as any to add modules to it. 

Hope that clears it up. 

Regards and happy new year!

Alex

**Comment by AlexDM0** [comment]
Hi,

Over the last year a lot of feature requests have been made. We have just introduced our new website which has a list of the requested features. We have placed this request on that list. 

The list can be found here:
http://visjs.org/featureRequests.html

An explaination of the new system can be found here:
http://visjs.org/blog.html#New\ website\ for\ vis\.js\!

I would like to stress that this does not mean we abandon this request. Discussion here will continue if needed on this feature but we will close it to keep our Github issue page more of a bug-todo list.

Future feature requests will still be made here and then added to the website by us.

Regards,

Alex
